### DISCUSSION TITLE
Failed to load module script

### DISCUSSION THREAD

**Discussion by sohailAhmeddev** [umputun/remark42#1728] [Q&A]
I'm trying to integrate remark42 in my next js app but was unable to do so.

<img width="960" alt="image" src="https://github.com/umputun/remark42/assets/56760407/1d5a30fd-223b-4899-8266-34fd8364c80b">

here is my configuration
    <script
        dangerouslySetInnerHTML={{
          __html: `
          var remark_config = {
            host: 'http://localhost:8080',
            site_id: 'remark',
            components:['embed']
          }`
        }}
      ></script>
        <script
        dangerouslySetInnerHTML={{
          __html: `
          !function (e, n) {
            for (var o = 0; o < e.length; o++) {
              var r = n.createElement("script"),
                c = ".js",  // Update the file extension to ".mjs"
                d = n.head || n.body;
              "noModule" in r ? (r.type = "module") : (r.async = !0),
                (r.defer = !0),
                (r.src = remark_config.host + "/web/" + e[o] + c),
                d.appendChild(r);
            }
          }(remark_config.components || ["embed"], document);`,
        }}
      ></script>

**Comment by paskal** [comment]
Does the Remark42 demo work for you on `http://<your host>:8080/web/`?

**Reply by sohailAhmeddev** [reply]
yes,

<img width="902" alt="image" src="https://github.com/umputun/remark42/assets/56760407/2e35530c-d704-4c8b-af45-01165172caaf">

**Reply by paskal** [reply]
It's not loaded correctly. Please check [working version](https://demo.remark42.com/web/) and compare what's different about yours.
